startLuminoWithTransport.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+### Usage
+
+## This script assumes:
+
+# You have NVM installed
+# You have NVM 14.13.1 installed
```

Comment:
isn't this `npm`?

Reply:
in fact is node..

maybe i can remove the nvm stuff and just assume that the user uses node 14.13.1

Reply:
latest version for `nvm` is `v0.37.2` so this can't be right

Reply:
its node version not nvm nor npm

Reply:
> its node version not nvm nor npm

it says `NVM`

Reply:
yep, fixed